src/solana_contrib/anchor.ts

```diff
@@ -5,22 +5,14 @@ import {
   EventParser,
   Idl,
   Instruction,
-  Wallet,
   utils,
 } from '@coral-xyz/anchor';
-import { InstructionDisplay } from '@coral-xyz/anchor/dist/cjs/coder/borsh/instruction';
-import { AllAccountsMap } from '@coral-xyz/anchor/dist/cjs/program/namespace/types';
-import {
-  AccountInfo,
-  Keypair,
-  PublicKey,
-  TransactionResponse,
-} from '@solana/web3.js';
+import type { InstructionDisplay } from '@coral-xyz/anchor/dist/cjs/coder/borsh/instruction';
+import type { AllAccountsMap } from '@coral-xyz/anchor/dist/cjs/program/namespace/types';
+import { AccountInfo, PublicKey, TransactionResponse } from '@solana/web3.js';
 import bs58 from 'bs58';
 import { ExtractedIx, extractAllIxs } from './transaction';
 
-export const dummyWallet = () => new Wallet(Keypair.generate());
-
```

Comment:
So I guess we moved this to tensor-infra?

Reply:
yup, the frontend complains that anchor doesn't export Wallet. It's exported in a funny way